### ISSUE TITLE
How to deserialize Option<T>

### ISSUE THREAD

**Issue by jonasbb** [softprops/envy#10]
Sorry, if the answer is obvious, but I could not find an example in the documentation nor the tests for this crate.

What should the content of an environment variable be if I want to deserialize a `Option<String>`? A non-existent variable results in `None`, but how to get a `Some(_)`?

**Comment by softprops** [comment]
Hi there @jonasbb. I was just going to put together a little demo then noticed that was a feature regressed. I'll try to get a fix out soon

**Comment by softprops** [comment]
Published fix in 0.3.1. Also added a bare bones example [here](https://github.com/softprops/envy/blob/master/examples/optional.rs)

```bash
$ cargo run --example optional
```

```bash
$ SIZE=1 cargo run --example optional
```

**Comment by softprops** [comment]
thanks for bringing up the issue!
